src/style.css

```diff
@@ -2,0 +1,114 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+:root {
+  --main-color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, Arial, sans-serif;
+  color: var(--main-color);
+  font-weight: 300;
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search--big {
+  height: 70px;
+  width: 100%;
+}
+
+.search--small {
+  height: 42px;
+  width: 100%;
+}
+
+.search__label {
+  position: relative;
+  width: 100%;
+  display: block;
+}
+
+.search__label::after {
+  content: '';
+  position: absolute;
+  display: block;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.search__label--big::after {
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.search__label--small::after {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.search__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+  color: var(--main-color);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
+  border-radius: 4px;
+  outline: none;
+}
+
+.search__input--big {
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.search__input--small {
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+.search__input::placeholder {
+  font-weight: 300;
+  color: var(--main-color);
+  font-family: Avenir, Arial, sans-serif;
+}
+
+.search__input--big::placeholder {
+  font-size: 16px;
+}
+
+.search__input--small::placeholder {
+  font-size: 14px;
+}
+
+.search__input:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search__input:focus {
+  border-radius: 3px;
+  font-weight: 900;
```

Comment:
This is a code quality issue. You're changing the `font-weight` on focus, which can cause a reflow and lead to a jarring effect for the user. It's generally best to keep consistent font styles on focus and simply highlight the text in some way.